src/importexport/mei/imeiconfiguration.h

```diff
@@ -37,6 +37,9 @@ class IMeiConfiguration : MODULE_EXPORT_INTERFACE
 
     virtual bool meiExportLayout() const = 0;
     virtual void setMeiExportLayout(bool value) = 0;
+
+    virtual bool meiUseMscoreIds() const = 0;
+    virtual void setMeiUseMscoreIds(bool value) = 0;
```

Comment:
I'm inclined to suggest not using `Mscore`, since it's an ugly abbreviation (and reminds of the `MScore` class, which is a very ugly one). I'd prefer writing `MuseScore` in full.